lua/lvim/core/log.lua

```diff
@@ -29,7 +29,7 @@ function Log:init()
           processors = {
             structlog.processors.Namer(),
             structlog.processors.StackWriter({ "line", "file" }, { max_parents = 0, stack_level = 2 }),
-            structlog.processors.Timestamper "%H:%M:%S",
+            structlog.processors.Timestamper "%Y/%m/%d %H:%M:%S",
```

Comment:
oh you actually modified the wrong one (console), it should be the one in `structlog.sinks.File` a few lines below 😄 

and I think it might be better to use  `"%F %H:%M:%S"` just it matches the one from neovim. https://github.com/neovim/neovim/blob/3d54fbf5c04dcb405e5c28ad7361273bf563ac92/runtime/lua/vim/lsp/log.lua#L17

Reply:
Whoops, my bad. What does the `%F` do ?

Reply:
If it is DDMonYY then I prefer G. I find the American order is terrible for proper sorting